packages/ui/src/Popup/AuthManagement/index.tsx

```diff
@@ -63,10 +59,14 @@ export function AuthManagement (): JSX.Element {
             Object.entries(authList as AuthUrls)
```

Comment:
I noticed that on the design the horizontal separation lines span all the way across. So one way to achieve this here could be to change the parent Box padding to only top and bottom `<Box py='s'` and in the `WebsiteEntry` component it could be wrapped with a Box with only left and right padding `<Box px='s'`